misc/healpix_levels.md

```diff
@@ -0,0 +1,56 @@
+# Towards a lookup list for HEALPIX levels
+
+![](healpix_global_indexing.png)
```

Comment:
We should mention that this is the `nested` HEALPIX indexing scheme

Reply:
feel free to push directly to #42, that's a modified version of the notebook you pushed here.